core/modules/cms/store/block/getters.ts

```diff
@@ -1,14 +1,35 @@
 import { GetterTree } from 'vuex'
 import CmsBlockState from '../../types/CmsBlockState'
 import RootState from '@vue-storefront/core/types/RootState'
+import { Logger } from '@vue-storefront/core/lib/logger'
 
 const getters: GetterTree<CmsBlockState, RootState> = {
-  cmsBlocks: (state) => state.items,
-  cmsBlockIdentifier: (state) => (identifier) => {
-    return state.items.find(item => typeof item === 'object' && item.identifier === identifier)
+  cmsBlocks: (state, getters) => {
+    Logger.error('The getter cmsBlock/addMethod has been deprecated please change to cmsBlock/getCmsBlocks')()
```

Comment:
unfortunately this is invoked not on usage but always in the application, even if not used. We should remove this getters and add instruction to upgrade notes to replace them to new ones around application.